### PR TITLE
Add Bijou.js w/ npm auto-update

### LINKED ISSUE #668: Add: Bijou.js
## Auto update mechanism
My auto update mechanism is NPM, the package source can be found [here](https://npmjs.com/bijou.js)

## Downloads per month / GitHub stars
My library has over 800 downloads per month on NPM:
image
image

## Other

**Library name:** Bijou.js

**Library license (include link):** [GNU General Public License v3.0](https://www.gnu.org/licenses/gpl-3.0.en.html)

**Library source (NPM package or Git repo):** [npm](https://npmjs.com/bijou.js), [github](https://github.com/bijou-js/bijou.js)

### PR DESCRIPTION
**Pull request by Explosion-Scratch**
I _think_ I did it right, I looked at the other PR's. Please tell me if I need to change something.

Resolves #668

### CODE REVIEW

**Comment by Explosion-Scratch** [comment]
@MattIPv4 Will you review this and merge? I'd be happy to fix stuff on it if needed!

**Comment by MattIPv4** [comment]
Will take a look when I have some time, maybe tomorrow 👍

**Comment by Explosion-Scratch** [comment]
> Will take a look when I have some time, maybe tomorrow 👍

Sure thing! Thanks! :D

File: packages/b/Bijou-JS.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Bijou.js",
```
Comment (MattIPv4) [inline packages/b/Bijou-JS.json:2]: The `name` property must match the filename (excl. `.json`), so this either needs to be `Bijou-JS`, or the file needs to be named to `Bijou.js.json` :)

File: packages/b/Bijou-JS.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "Useful JS snippets, in one simple library",
+  "keywords": [
+    "bijou",
+    "javascript",
+    "library",
+    "functions",
+    "useful",
+    "elegant"
+  ],
+  "license":"GNU General Public License v3.0",
```
Comment (MattIPv4) [inline packages/b/Bijou-JS.json:12]:
```suggestion
  "license": "GNU General Public License v3.0",
```
Spacing

File: packages/b/Bijou-JS.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "homepage":"https://bijou.js.org",
```
Comment (MattIPv4) [inline packages/b/Bijou-JS.json:13]:
```suggestion
  "homepage": "https://bijou.js.org",
```
Spacing

File: packages/b/Bijou-JS.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  "authors": [
+    {
+      "name": "Explosion Implosion",
+      "email": "explosionscratch@gmail.com",
+      "url": "https://github.com/explosion-scratch"
+    }
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target":"bijou.js",
```
Comment (MattIPv4) [inline packages/b/Bijou-JS.json:27]:
```suggestion
    "target": "bijou.js",
```
Spacing

File: packages/b/Bijou-JS.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    "fileMap":[
+      {
+        "basePath":"/",
```
Comment (MattIPv4) [inline packages/b/Bijou-JS.json:30]:
```suggestion
        "basePath": "",
```
Base path can just be blank if getting files from the root of the project (+ spacing)

File: packages/b/Bijou-JS.json [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        "files":["bijou.js"]
```
Comment (MattIPv4) [inline packages/b/Bijou-JS.json:31]:
```suggestion
        "files": ["bijou.js"]
```
Spacing

---

Do we also want `bijou.css` and `bijuo-min.js`?

File: packages/b/Bijou-JS.json [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      }
+    ]
+  },
+  "filename":"bijou.js"
```
Comment (MattIPv4) [inline packages/b/Bijou-JS.json:35]:
```suggestion
  "filename": "bijou.js"
```
Spacing

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Ty for the PR, sorry for the delay!

File: packages/b/Bijou-JS.json (hunk 6 above)
Comment (Explosion-Scratch) [reply packages/b/Bijou-JS.json]: Yes! I'll add those!

File: packages/b/Bijou-JS.json (hunk 1 above)
Comment (Explosion-Scratch) [reply packages/b/Bijou-JS.json]: Done! It's `Bijou.js.json` now! (I didn't want to do that originally because I was worried it'd count as a JS file)

**Comment by Explosion-Scratch** [comment]
^^
I also renamed it in that commit.

**Comment by Explosion-Scratch** [comment]
@MattIPv4 Will you re-review?

**Review by MattIPv4 — APPROVED** [review]
Lgtm, tyvm!

**Pull request merged**